commit 18908364dac05b77c4b376e181effdaa7f6c11f4 (refs/changes/62/1162/3)
Author: Dillon Amburgey <dillona@dillona.com>
Date:   2016-12-29T19:47:27+00:00 (2 years, 9 months ago)
    
    joyent/manta-thoth#146 Broken link in readme
    Reviewed by: Bryan Cantrill <bryan@joyent.com>

diff --git a/README.md b/README.md
index 35acf11..6ad74bb 100644
--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ further data movement.
 
 # Setup
 
-As with the [node-manta](http://github.com/node-manta) CLI tools, you will
+As with the [node-manta](https://github.com/joyent/node-manta) CLI tools, you will
 need to set Manta environment variables that match your Joyent Manta account:
 
     $ export MANTA_KEY_ID=`ssh-keygen -l -f ~/.ssh/id_rsa.pub | awk '{print $2}' | tr -d '\n'`
